### PR TITLE
Updates mtu configuration to use neutron config options.

### PR DESCRIPTION
**Pull request by rossella**
Removes custom dnsmasq-neutron.conf.erb and sets mtu options via.
neutron ml2_conf.ini.erb.

### CODE REVIEW

**Comment by rossella** [comment]
WARN I haven't tested this

**Comment by dirkmueller** [comment]
+1

**Comment by aplanas** [comment]
+1

File: chef/cookbooks/neutron/templates/default/dhcp_agent.ini.erb [hunk 1]
```diff
@@ -80,10 +80,6 @@ enable_metadata_network = <%= @enable_metadata_network %>
 # dhcp_domain = openstacklocal
 dhcp_domain = <%= @dhcp_domain %>
 
-# Override the default dnsmasq settings with this file
-# dnsmasq_config_file =
-dnsmasq_config_file = /etc/neutron/dnsmasq-neutron.conf
```
Comment (vuntz) [inline chef/cookbooks/neutron/templates/default/dhcp_agent.ini.erb]: I think we should keep the comment and the default (commented out) value, to not diverge from the default configuration file. So just remove that third line.

**Comment by mmnelemane** [comment]
+1

File: chef/cookbooks/neutron/templates/default/dhcp_agent.ini.erb (hunk 1 above)
Comment (rossella) [inline chef/cookbooks/neutron/templates/default/dhcp_agent.ini.erb]: agreed, thanks

**Pull request merged**